_data/events/Pune.json

```diff
@@ -0,0 +1,18 @@
+{
+    "title": "GDCR17 Nelkinda, Pune",
+    "url": "http://nelkinda.com/events/2017/11/18/Global-Day-of-Coderetreat/pune-nelkinda",
+    "moderators": [
+        "@SiddheshNikude",
+        "@nelkinda"
```

Comment:
Please move the sponsors to the [`sponsors`](https://github.com/coderetreat/coderetreat.github.io/blob/master/events/event_schema.json#L61-L67) field